### PR TITLE
Add botor

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@benhuff @rluria14 agree to maintain?

**Comment by oblute** [comment]
@conda-forge/staged-recipes this recipe is ready for review

File: recipes/botor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - boto3
+    - python
+
+test:
+  imports:
+    - botor
+    - botor.aws
+
+about:
+  home: https://github.com/monkeysecurity/botor
```
Comment (npavlovikj) [inline recipes/botor/meta.yaml]: This url doesn't exist...

File: recipes/botor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "botor" %}
+{% set version = "0.0.1dev8" %}
```
Comment (npavlovikj) [inline recipes/botor/meta.yaml]: Is this an official, or pre-release package? If the latter, you will probably need to use the `conda-forge prerelease` target instead.

Comment (oblute) [reply recipes/botor/meta.yaml]: @npavlovikj this is a pre-release package, available for download from an official source, PyPi.  I think that the target tag must be added to the conda-forge.yml after the feedstock is created, like done here with the "dev" target: https://github.com/conda-forge/sense2vec-feedstock/blob/master/conda-forge.yml

Comment (npavlovikj) [reply recipes/botor/meta.yaml]: You can actually add the `conda-forge.yml` file here, as in https://github.com/conda-forge/staged-recipes/pull/10497/files.

Comment (oblute) [reply recipes/botor/meta.yaml]: Thank you - I added it as in that example you linked

**Comment by npavlovikj** [comment]
The recipe looks good to me @oblute , thank you for contributing 🎉

**Pull request merged**